### DISCUSSION TITLE
PotatoAlert installation

### DISCUSSION THREAD

**Discussion by KenrodMelrocity** [razaqq/PotatoAlert#46] [Q&A]
Hello - is there a readme document with simple step-wise instructions for potatoalert install? I have used it for years. When I opened it yesterday I clicked 'yes' when it asked me if I wanted to install an update. At that point it wouldn't work anymore. I've tried and failed multiple times to reinstall. Would appreciate some help if you don't mind. FYI I am not all that computer savvy, at least not to the extent I see here on github. Thank you

**Comment by razaqq** [comment]
1. Download the archive from https://github.com/razaqq/PotatoAlert/releases/tag/3.1.10
2. Unpack anywhere
3. Start it

**Reply by KenrodMelrocity** [reply]
Thanks. I think that is what I'm doing but it's not working.

1. I download PotatoAlert.zip from the link you provided
2. I extract all from the zip file
3. Inside the unpacked file, double click to open the Application file (14,270 kb) titled PotatoAlert with the potato icon.
4. A blue box opens title "Windows protected your PC". I click the "Run anyway" button.
5. The blue box disappears but the app is not installed, and no shortcut is added to my desktop.

Thanks for the quick response, much appreciated.

**Reply by razaqq** [reply]
Are you on windows 7 or 10?

You did everything correct, but it should just start, no installing, no shortcut.
Im assuming it does not start then?
Were you using a previous version in the past?

**Comment by KenrodMelrocity** [comment]
Windows 10. Been using 10 for a long time. PA worked fine up until
yesterday when I clicked "yes" when potatoalert asked if I wanted to update.

**Reply by razaqq** [reply]
Can you post your config `C:\Users\MYUSERNAME\AppData\Local\PotatoAlert\config.json`?

**Comment by KenrodMelrocity** [comment]
I see that file but cannot open it, it's asking me to find the app. Copy is
attached.

**Reply by razaqq** [reply]
You can open the file with notepad, notepad++ or similar.
I can unfortunately not see an attachment when you reply to github.

**Comment by KenrodMelrocity** [comment]
Somehow between this and updating Aslains my WOWS folder has been screwed
up, need to reinstall the game from scratch.... (sigh)

**Reply by razaqq** [reply]
PotatoAlert never modifies any files from wows, this has to be related to aslains

**Comment by KenrodMelrocity** [comment]
Thanks, opened with Notepad, results below. (must have been Aslains, I had
just updated it)

{
    "game_directory": "C:/Games/World_of_Warships",
    "language": 0,
    "match_history": true,
    "menubar_left": true,
    "minimize_tray": false,
    "override_replays_directory": false,
    "replays_directory": "C:/Games/World_of_Warships/replays",
    "save_match_csv": false,
    "stats_mode": 0,
    "table_layout": "horizontal",
    "team_damage_mode": "average",
    "team_win_rate_mode": "average",
    "update_notifications": true,
    "window_height": 1040,
    "window_state": 2,
    "window_width": 1920,
    "window_x": 664,
    "window_y": 166
}

**Reply by razaqq** [reply]
I see the problem, change `"stats_mode": 0,` to `"stats_mode": "pvp",` for now to fix it.
I will fix this in the next update.

**Comment by KenrodMelrocity** [comment]
Yep, that fixed it. Thank you!!
